manager/controllers/motion/streamtransfer_controller.go

```diff
@@ -6,6 +6,7 @@ package motion
 import (
 	"context"
 	"fmt"
+
```

Comment:
your local path for goimports is probably wrong. `k8s.io` should be separated

Reply:
When I run goimports again now I do see some changes like this one. Probably had a typo with repo's name.
I'll had another commit to fix it.